packages/sdk/src/chains/EVM/helpers.ts

```diff
@@ -26,19 +28,22 @@ export const addPadding = (covertThis: string | number, padding: number): string
 };
 
 /**
- * @name createResourceID
- * @description creates a resource id based on the contract address
+ * Creates a resource id based on the contract address
+ *
  * @param contractAddress
  * @param domainID
  * @returns {string}
  */
-export const createResourceID = (contractAddress: string, domainID: number): string => {
-  return toHex(contractAddress + toHex(domainID, 1).substr(2), 32);
+export const createResourceID = (contractAddress: string, domainID: string | number): string => {
```

Comment:
I don't understand the use case for this method, as we are not defining resource ID based on some logic like this.

Reply:
it was copied from solidity helpers. I've removed it as unused